src/main/java/org/traccar/geolocation/OpenCellIdGeolocationProvider.java

```diff
@@ -48,8 +48,14 @@ public void completed(JsonObject json) {
                         callback.onSuccess(
                                 json.getJsonNumber("lat").doubleValue(),
                                 json.getJsonNumber("lon").doubleValue(), 0);
+                    } else if (json.containsKey("error")) {
+                        String errorMessage = json.getJsonString("error").getString();
+                        if (json.containsKey("code")) {
+                            errorMessage += " Error code: " + json.getInt("code");
```

Comment:
I would recommend including error just in brackets without a label. Otherwise we are risking localization mismatch between original error and this label. I mean something like this:
```
Cell not found (1)
```